topology/mask.h

```diff
@@ -312,43 +312,48 @@ class BallMask : public Mask< D >
 };
 
 /**
- * Mask defining a elliptical region. Only 2D is implemented
+ * Mask defining an elliptical or ellipsoidal region.
  */
 template < int D >
 class EllipseMask : public Mask< D >
 {
 public:
   /**
    * @param center Center of ellipse
-   * @param major_axis Length between center and vertex on major axis
-   * @param minor_axis Length between center and vertex on minor axis
-   * @param intermediate_axis Length between center and vertex on z-axis
-   * @param angle Angle between x-axis and major axis of the ellipse
+   * @param major_axis Length of major axis of ellipse or ellipsoid
+   * @param minor_axis Length of minor axis of ellipse or ellipsoid
+   * @param polar_axis Length of polar axis of ellipsoid
+   * @param azimuth_angle Angle between x-axis and major axis of the ellipse or
+   *        ellipsoid
+   * @param polar_angle Angle between z-axis and polar axis of the ellipsoid
```

Comment:
After looking at sinusoidal stimulus generators, where we specify phase in degrees, I think we should specify angles in degree here as well, and then convert to radians below.